config/application.rb

```diff
@@ -22,5 +22,21 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    config.action_mailer.perform_deliveries = true
+    config.action_controller.perform_caching = false
+
+    # Don't care if the mailer can't send.
+    config.action_mailer.perform_deliveries = true
+    config.action_mailer.raise_delivery_errors = true
+    config.action_mailer.delivery_method = :smtp
+    config.action_mailer.smtp_settings = {
+      :port           => 587,
+      :address        => 'smtp.mailgun.org',
+      :user_name      => 'postmaster@sandbox85b185dc6fa44a7bbb05e2ce19beca69.mailgun.org',
+      :password       => '065168f69b0027f0cb9f05b917383d0a',
+      :domain         => 'sandbox85b185dc6fa44a7bbb05e2ce19beca69.mailgun.org',
+      :authentication => :plain
+    }
```

Comment:
you might consider moving this mailer code into its own config file and also using environment variables or something else to protect your mailer credentials